### PR TITLE
Rebeca lopez santos/this week button

### PR DESCRIPTION
**Pull request by rebecalsantos**
Updated to show this week by default and taking to current week when clicking 'This Week' button

### CODE REVIEW

File: client/src/pages/WeeklyTable.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-	"2022-01-20T00:00:00.000Z",
-	"2022-01-21T00:00:00.000Z",
-];
+// const week = [
+// 	"2022-01-17T00:00:00.000Z",
+// 	"2022-01-18T00:00:00.000Z",
+// 	"2022-01-19T00:00:00.000Z",
+// 	"2022-01-20T00:00:00.000Z",
+// 	"2022-01-21T00:00:00.000Z",
+// ];
```
Comment (teomrd) [inline client/src/pages/WeeklyTable.js]: Why not deleting the comments to keep the code-base cleaner?

File: client/src/pages/WeeklyTable.js [hunk 2]
```diff
@@ -109,7 +113,9 @@ const WeeklyTable = ({ bookings, desks, refreshBooking, maxDesksForDay }) => {
 		<div>
 			<div>
 				{/* <tr> */}
```
Comment (teomrd) [inline client/src/pages/WeeklyTable.js]: Here as well

**Review by teomrd — APPROVED** [review]
Looks good! I would only remove the comments, there is no reason of keeping the in production code, as with git we can always retrieve them back if needed.

File: client/src/pages/WeeklyTable.js (hunk 2 above)
Comment (rebecalsantos) [reply client/src/pages/WeeklyTable.js]: done, many thanks for reviewing

File: client/src/pages/WeeklyTable.js (hunk 1 above)
Comment (rebecalsantos) [reply client/src/pages/WeeklyTable.js]: done, many thanks for reviewing

**Comment by rebecalsantos** [comment]
Thanks for the review Theo and the help

**Pull request merged**